lib/sms.rb

```diff
@@ -0,0 +1,22 @@
+require 'dotenv'
+Dotenv.load
+require 'twilio-ruby'
+require "./lib/order.rb"
+
+class SMS
+
+  def send_sms(message)
```

Comment:
Could this method be broken down to make it more readable? To understand it at the moment, you need to read the steps and put them together in your head, rather than reading it as a set of higher level things that are being done.